content/influxdb/cloud-dedicated/reference/influxql/offset-and-soffset.md

```diff
@@ -0,0 +1,249 @@
+---
+title: OFFSET and SOFFSET clauses
+description: >
+  Use `OFFSET` to specify the number of [rows](/influxdb/cloud-dedicated/reference/glossary/#series)
+  to skip in each InfluxQL group before returning results.
+  Use `SOFFSET` to specify the number of [series](/influxdb/cloud-dedicated/reference/glossary/#series)
+  to skip before returning results.
+menu:
+  influxdb_cloud_dedicated:
+    name: OFFSET and SOFFSET clauses
+    parent: influxql-reference
+weight: 207
+list_code_example: |
+  ```sql
+  SELECT_clause FROM_clause [WHERE_clause] [GROUP_BY_clause] [ORDER_BY_clause] [LIMIT_clause] OFFSET row_N [SLIMIT_clause] SOFFSET series_N
+  ```
+---
+
+Use `OFFSET` to specify the number of [rows](/influxdb/cloud-dedicated/reference/glossary/#series)
+to skip in each InfluxQL group before returning results.
+Use `SOFFSET` to specify the number of [series](/influxdb/cloud-dedicated/reference/glossary/#series)
+to skip before returning results.
+
+  - [OFFSET clause](#offset-clause)
+     - [Syntax](#offset-syntax)
+     - [Notable OFFSET clause behaviors](#notable-offset-clause-behaviors)
+     - [Examples](#offset-examples)
+  - [SOFFSET clause](#soffset-clause)
+     <!-- - [Syntax](#soffset-syntax) -->
+     <!-- - [Notable SOFFSET clause behaviors](#notable-soffset-clause-behaviors) -->
+     <!-- - [Examples](#soffset-examples) -->
+
+## `OFFSET` clause
+
+The `OFFSET` clause skips `N` rows in each InfluxQL group before returning results.
+Offsets honor row limits specified in the
+[`LIMIT` clause](/influxdb/cloud-dedicated/reference/influxql/limit-and-slimit/#limit-clause)
+and display the limited number of rows after the specified offset.
+Use `LIMIT` and `OFFSET` together to paginate query results.
+
+### Syntax {#offset-syntax}
+
+```sql
+SELECT_clause FROM_clause [WHERE_clause] [GROUP_BY_clause] [ORDER_BY_clause] [LIMIT_clause] OFFSET N [SLIMIT_clause] [SOFFSET_clause]
+```
+
+#### Arguments 
+
+- **N**: Number of rows to skip in each InfluxQL group before returning results. 
+
+### Notable OFFSET clause behaviors
+
+- If there is no `LIMIT` clause in a query with an `OFFSET` clause, InfluxQL
+  assumes the behavior, `LIMIT 1`.
```

Comment:
This is not accurate – it assumes no `LIMIT` clause, which I verified with OSS and also how it is implemented in IOx.

Reply:
Interesting. In my testing, if I included an offset without a limit, it would return a single row per group at the correct offset. Is that the expected behavior?